src/canvg.js

```diff
@@ -419,11 +419,22 @@ function build(opts) {
     if (s.match(/cm$/)) return this.numValue() * this.getDPI(viewPort) / 2.54;
     if (s.match(/mm$/)) return this.numValue() * this.getDPI(viewPort) / 25.4;
     if (s.match(/in$/)) return this.numValue() * this.getDPI(viewPort);
-    if (s.match(/%$/)) return this.numValue() * svg.ViewPort.ComputeSize(viewPort);
+    if (s.match(/%$/)) {
```

Comment:
this part looks fine, you can always do `return this.numValue() * (considerParentFontSize ? this.getEM(viewPort) : svg.ViewPort.ComputeSize(viewPort);` for brevity. optional for this PR